src/components/form/input.js

```diff
@@ -201,17 +205,20 @@ const Input = ({
                     error={error}
                     disabled={disabled}
                     height={height}
+                    showLabel={label}
```

Comment:
This will have a global effect, will it affect any of the other form inputs elsewhere?